framework/validators/InlineValidator.php

```diff
@@ -100,7 +100,7 @@ public function clientValidateAttribute($model, $attribute, $view)
             if ($current === null) {
                 $current = $model->$attribute;
             }
-            return $method($attribute, $this->params, $this, $current);
+            return $method($attribute, $this->params, $this, $current, $view);
```

Comment:
phpdoc for `$clientValidate` property should be adjusted to mention `$view` in its signature. `$current` parameter seems to be missing as well in the example.

Reply:
@samdark done